### ISSUE TITLE
Wrong work search 

### ISSUE THREAD

**Issue by dimona1** [ONLYOFFICE/CommunityServer#172]
After update from v9 to v10, search work only with new data, which was recieved after update. Old data don't find in email, and other modules.

**Comment by Carazyda** [comment]
Hello @dimona1  Please try to reindex your data, for this you need to remove old index and restart our services.
`sudo service god stop`
`sudo service onlyofficeIndex stop`
`sudo service elasticsearch stop`
`rm -rf /var/www/onlyoffice/Data/Index/*`
`mysql -u root -p -D onlyoffice`
`delete from webstudio_index;`
`sudo service elasticsearch start`
`sudo service onlyofficeIndex start`
`sudo service god start`

**Comment by dimona1** [comment]
Hello!
It didn't help for me
Now I lost data and from update ))

**Comment by Carazyda** [comment]
Indexing can take a long time, depending on the amount of your data. Please try search a little later.

**Comment by dimona1** [comment]
I waited more than 20 hours, but the data did not appear

**Comment by Carazyda** [comment]
Did you delete the index yourself before my comment? Completely deleted as I described and restarted the services?

**Comment by dimona1** [comment]
Oh, i didn't remove /var/www/onlyoffice/Data/Index/*
Now I have done it, I will wait for the update

**Comment by dimona1** [comment]
> Did you delete the index yourself before my comment? Completely deleted as I described and restarted the services?

After the indexes were completely removed, the situation did not change. In the search, the data obtained after deleting the indexes appears

**Comment by Carazyda** [comment]
No errors when executing commands that i described? Maybe any errors in the svcIndex.log or elasticsearch.log?

**Comment by dimona1** [comment]
@Carazyda, have you any idea?

**Comment by Carazyda** [comment]
Other services are work well? No errors with mysql in their logs? Please specify what type installation do you use, OS? Have you had any problems with update from v9 to v10?

**Comment by dimona1** [comment]
And so, I solved the problem.
The main problem was installed mysql v 5.5, was installed on my server. Therefore, an error appeared in mysql logs:
`ERROR [32] ASC.Indexer - MySql.Data.MySqlClient.MySqlException (0x80004005): FUNCTION onlyoffice.JSON_ARRAY does not exist`


After updating mysql, the search still did not work even after deleting indexes. Found that after running elasticsearch, and onlyofficeIndex, an error appears in the logs:
`[DEBUG] [o.e.a.b.TransportBulkAction] [KE6g1AE] failed to execute pipeline [attachments] for document [mail_mail / mailwrapper / 356012]`

The problem was resolved by delaying the launch of onlyofficeIndex until elasticsearh was fully launched.

@Carazyda Thanks for the help.
